index.bs

```diff
@@ -1450,6 +1451,60 @@ that are returned to the caller when a new credential is created, or a new asser
         [=extension identifier=] → [=client extension output=] entries produced by the extension's
         [=client extension processing=].
 
+    :   {{PublicKeyCredential/toJSON()}}
+    ::  This operation returns {{RegistrationResponseJSON}} or {{AuthenticationResponseJSON}}, that is a JSON-serialized
```

Comment:
I'm not sure if this is accurate. JSON-serialized implies string, right? Perhaps the entire sentence could be deleted?

Reply:
> JSON-serialized implies string, right?

Does it? If you tell a JS dev "this outputs JSON" I'll bet they're going to think they'll get back an `object` instead of a `string`. If what you're saying is true then I'll need to adjust this wording to reflect that the output should be an `object`. Something like:

> This operation returns...an `object` that can be fed to `JSON.stringify()` to get back a string that can be sent to the RP as `"application/json"` without mutilating any `ArrayBuffer`/`Uint8Array`'s that might be present in the original `object`.

Reply:
So the `toJSON` spec says the method returns a "JSON type" (object, array, string, number, etc.) and calls the output of JSON.stringify a "JSON string". Perhaps we can reuse the established language? 

> This operation converts the `PublicKeyCredential` into a `PublicKeyCredentialJSON`, which is a [JSON type](https://webidl.spec.whatwg.org/#dfn-json-types). In ECMAScript, instances of `PublicKeyCredentialJSON` can be converted into a JSON string with the [JSON.stringify() method](https://262.ecma-international.org/12.0/#sec-json.stringify).

Reply:
@emlun helped craft verbiage below that reflected @kreichgauer's suggestion to match established language.